packages/node/src/node.ts

```diff
@@ -40,7 +45,7 @@ export class Node {
 
   private readonly ioSendDeferrals = new Map<
     string,
-    Deferred<NodeMessageWrappedProtocolMessage>
+    Deferred<NodeMessageWrappedProtocolMessage | NodeMessageWrappedFinMessage>
```

Comment:
the typing seems like it could be combined into a declared union type within the types file, but if were only using it here it doesn't really make too much of a difference. something to keep in mind if we end up using this in other places though